corehq/apps/app_manager/static/app_manager/js/download_index_main.js

```diff
@@ -21,6 +21,13 @@ hqDefine('app_manager/js/download_index_main',[
                 initialPageData.reverse("get_multimedia_sizes", appID, 'default'));
             multimediaSizes.load();
             $("#multimedia-sizes-container").koApplyBindings(multimediaSizes);
+            if ($('#build-profile-select-for-multimedia').length) {
```

Comment:
fyi, this if isn't necessary, calling `on` will just do nothing if `#build-profile-select-for-multimedia` isn't found.

Reply:
Ya, was avoiding setting up the view completely as called above, so kept this 'on' within the if block only

Reply:
oh sorry, misinterpreted. Removing it

Reply:
https://github.com/dimagi/commcare-hq/pull/26032/commits/3463c71b7f75a8df3ccc00e4424ea632ea3f6e0b